python-base/ubuntu22.04-python3.10/docker/scripts/install_python.sh

```diff
@@ -6,10 +6,12 @@ set -exuo pipefail
 # Install python
 apt-get install -y --no-install-recommends \
   "python${PYTHON_VERSION}" \
+  "python${PYTHON_VERSION}-venv" \
   python3-distutils
 
 # Set python active
 update-alternatives --install /usr/bin/python python "/usr/bin/python${PYTHON_VERSION}" 10
+update-alternatives --install /usr/bin/python3 python3 "/usr/bin/python${PYTHON_VERSION}" 10
```

Comment:
I'm pretty sure there was a reason this wasn't done, Ubuntu sucks, and a lot of built-in utilities specifically say `#!/usr/bin/python3` and they all break if you do this.

Reply:
I couldn't at least notice anything breaking, but of course didn't do any too thorough analysis of other tools in the image. I can try to revert this change, but I'm afraid that will mean poetry might get installed by some other python interpreter (I think they were using `python3`). But even if poetry gets installed/run by some other version it should be possible to configure it to use the active interpreter when creating the environments. I will need to check and get back.

Reply:
Yes, it's not easy to notice, because it's random utilities in random situations that randomly blow up because their incorrectly installed system-wide dependencies are installed in some location where the package manager installed default Python can find them but no other Python version can find them. I've had various `apt-*` randomly crash with impossible to solve issues when making this change on multiple occasions.

Reply:
All thanks to Ubuntu and most other Linux distros not knowing how to build a distro in a sane manner and instead depending on having fixed versions for things the users will want to customize. 🥳

Reply:
Okay, seems like my test with removing this line (and thus reverting back to the situation before this PR) still produced a working image and the environment created by poetry still uses python 3.10 as expected.